content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -54,22 +54,26 @@ Check on the status of the Job with `kubectl`:
 
 {{< tabs name="Check status of Job" >}}
 {{< tab name="kubectl describe job pi" codelang="bash" >}}
-Name:             pi
-Namespace:        default
-Selector:         controller-uid=0cd26dd5-88a2-4a5f-a203-ea19a1d5d578
-Labels:           controller-uid=0cd26dd5-88a2-4a5f-a203-ea19a1d5d578
-                  job-name=pi
-Annotations:      batch.kubernetes.io/job-tracking: 
-Parallelism:      1
-Completions:      1
-Completion Mode:  NonIndexed
-Start Time:       Fri, 28 Oct 2022 13:05:18 +0530
-Completed At:     Fri, 28 Oct 2022 13:05:21 +0530
-Duration:         3s
-Pods Statuses:    0 Active / 1 Succeeded / 0 Failed
+Name:           pi
+Namespace:      default
+Selector:       batch.kubernetes.io/controller-uid=c9948307-e56d-4b5d-8302-ae2d7b7da67c
+Labels:         batch.kubernetes.io/controller-uid=c9948307-e56d-4b5d-8302-ae2d7b7da67c
+                batch.kubernetes.io/job-name=pi
+                job-name=pi
+                controller-uid=c9948307-e56d-4b5d-8302-ae2d7b7da67c
```

Comment:
I know these currently show, but since they might eventually delete them, maybe we can remove it from the docs?
Hopefully to reduce their usage

Reply:
I can see that.  Would it cause confusion if someone does `kubectl describe job pi` and the results are different than from the example?

I would vote to have the documentation be the same as what a user would get?

Reply:
You can just showcase the one that are relevant and "hide" the ones we want to hide behind `...`  This way we'd "push" users towards using the new labels, still being honest with contents in the docs :wink: 

In general, I agree with Aldo, it's better to push users towards the correct labels than keeping the old deprecated ones. If you look at kubectl, for example, all the deprecated commands are not showing in help. This is similar case, imo.

Reply:
Thank you for the clarification. I’ll update this.

Reply:
Done.